rules.txt

```diff
@@ -1096,7 +1096,7 @@
 /ig(.*)                                   301  /departments/office-of-the-inspector-general/
 /ima/                                     301  /departments/office-of-immigrant-affairs/
 /immigrant-business-week                  301  /spotlight/immigrant-business-week-2020/
-/immigrant-?heritage-?month                 301  /spotlight/immigrant-heritage-month-2024/
+/immigrant-heritage-month                 301  /spotlight/immigrant-heritage-month 
```

Comment:
Here you want to preserve the ? regexes in the URL to redirect, since it's matching on that url to redirect people. The destination URL is correct, though!